### PR TITLE
Create 2023-may-intro

### PR DESCRIPTION
**Pull request by aldenc**
created list of participant names for demo

### Summary

Fixes #<NUM>

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### List of changes proposed in this PR (pull-request)

* *Lorem ipsum dolor sit amet, consectetur adipiscing.*
* *Lorem ipsum dolor sit amet, consectetur adipiscing.*

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: workshops/github-workshop/2023-may-intro [hunk 1]
```diff
@@ -0,0 +1,5 @@
+1. Manjeet
+2. Emma
```
Comment (Arielle-Bennett) [inline workshops/github-workshop/2023-may-intro:2]:
```suggestion
2. Emma Chapman
```

File: workshops/github-workshop/2023-may-intro
Comment (Arielle-Bennett) [inline workshops/github-workshop/2023-may-intro:1]: Note that this file needs the suffix ".md" to be valid

**Review by Arielle-Bennett — CHANGES_REQUESTED** [review]
Please add suffix to file before merging!

File: workshops/github-workshop/2023-may-intro
Comment (aldenc) [reply workshops/github-workshop/2023-may-intro:1]: done

**Pull request merged**